lib/regex.js

```diff
@@ -3,7 +3,7 @@
 module.exports = Object.freeze({
     CLASS_REGEX: /(class|interface|trait)\s+(\w+)\s?/,
     PROPERTY_REGEX: /(\s+)?(?:public|protected|private)\s+(?:static\s+)?((?:\??[\w\\]+)\s+)?(\$\w+)(?:\s*=\s*([^;]+))?;/,
-    METHOD_REGEX: /(\s+)?(?:(?:abstract\s+)?(?:public|private|protected)?\s+(?:static\s+)?)?function\s+(\w+)\s*\((.+)?\)(?:\s*:\s*(\??[\w\\]+))?/,
+    METHOD_REGEX: /(\s+)?(?:[abstract|public|private|protected|static]+\s+)*function\s+(\w+)\s*\((.+)?\)(?:\s*:\s*(\??[\w\\]+))?/,
```

Comment:
```suggestion
    METHOD_REGEX: /(\s+)?(?:(?:abstract|public|private|protected|static)\s)*function\s+(\w+)\s*\((.+)?\)(?:\s*:\s*(\??[\w\\]+))?/,
```

Reply:
Yes, ```abstractpublicstaticprotected function wtf()``` and ```public private function``` are not valid methods. PHP compiler and PHP IDE shows it.
I think the extension should not check PHP syntax. 

I fixed problem for ```abstractpublicstaticprotected function wtf()```.